src/util/std_code.h

```diff
@@ -139,7 +139,9 @@ class code_blockt:public codet
 
   void move(codet &code)
   {
-    move_to_operands(code);
+    code_operandst &stmts = statements();
```

Comment:
This seems obviously worse. Given move semantics are fairly widely used in the codebase and not deprecated, and `irept::swap` is used to fake move semantics, why not just remove the tag from `move_to_operands`?

Reply:
What about making `move` take an rvalue reference? (But also I think getting rid of `exprt::move_to_operands` in here is actually a good idea, because eventually a `codet` should no longer be an `exprt`.)